core/embed/rust/src/micropython/print.rs

```diff
@@ -0,0 +1,10 @@
+use super::ffi;
+use heapless::String;
+
+/// Print a String into emulator console. Is being '\0' terminated automatically.
+pub fn print<const N: usize>(mut to_log: String<N>) {
+    to_log.push('\0').unwrap();
+    unsafe {
+        ffi::mp_print_str(&ffi::mp_plat_print, to_log.as_ptr() as _);
+    }
+}
```

Comment:
i found a better function: `mp_print_strn`, which accepts `len`.
```suggestion
pub fn print(to_log: &str) {
    unsafe {
        ffi::mp_print_strn(
            &ffi::mp_plat_print,
            to_log.as_ptr() as _,
            to_log.len(),
            0,
            '\0',
            0,
        );
    }
}
```